docs/deploy-configure.md

```diff
@@ -121,6 +134,65 @@ To scale a replica set:
    might take several minutes to remove the StatefulSet replicas for the
    members that you remove from the replica set.
 
+## Add Arbiters to a Replica Set
+
+You can add [arbiters](https://docs.mongodb.com/manual/core/replica-set-arbiter/) to your replica set by adding the `spec.arbiters` field to your MongoDB resource definition. 
+
+The value of the `spec.arbiters` field must be:
+
+- a positive integer, and
+- less than the value of the `spec.members` field. At least one replica set member must not be an arbiter.
```

Comment:
**Suggestion:** This last sentence looks a little awkward as a continued thought to the requirement. I suggest making the sentence  its own line or maybe a note.

```suggestion
- less than the value of the `spec.members` field.
  At least one replica set member must not be an arbiter.
```

Reply:
I had this as a note in an earlier draft. Moved it back out to a note.